frontend/model/database.js

```diff
@@ -34,14 +165,17 @@ class EmptyValue extends Error {}
 export const SETTING_CURRENT_USER = '@settings/currentUser'
 
 sbp('sbp/selectors/register', {
+  'gi.db/ready': function () {
+    return localforage.ready()
+  },
   'gi.db/settings/save': function (user: string, value: any): Promise<*> {
-    return appSettings.setItem(user, value)
+    return appSettings.setItem('u' + user, value)
```

Comment:
What's the reason for this `u` prefix? (Can you add a comment?)

Reply:
It's to disambiguate the type of data that's being stored. I can add a comment.